R/pkg/R/utils.R

```diff
@@ -624,15 +624,22 @@ convertNamedListToEnv <- function(namedList) {
   env
 }
 
-# Assign a new environment for attach() and with() methods
-assignNewEnv <- function(data) {
-  stopifnot(class(data) == "DataFrame")
-  cols <- columns(data)
-  stopifnot(length(cols) > 0)
-
+# Put each column of a DataFrame into a new environment
```

Comment:
This function is actually copying all columns to one new env. 'each' is not clear.